src/Zoit/zoit/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for zoit project.
+
+Generated by 'django-admin startproject' using Django 3.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+
+from django.conf.global_settings import FIXTURE_DIRS
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '!3q)zx_!$7z$0j2yy4^7+0t@4#thmjjrh_s9j+#1fl19fzd6vz'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'Zoitapp.apps.ZoitappConfig',
```

Comment:
use the name used in the class, i.e. name='Zoitapp', and not the classname for clarity
so here it should be:
INSTALLED_APPS = [
 'Zoitapp',
...,
]
Preferably add it at the end of the dictionary.